app/code/Dss/Easypathhints/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<!--
+/**
+* Digit Software Solutions.
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the EULA
+* that is bundled with this package in the file LICENSE.txt.
+*
+* @category  Dss
+* @package   Dss_Easypathhints
+* @author    Extension Team
+* @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+*/
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="dss" translate="label" sortOrder="200">
```

Comment:
again why used <tab id="dss" translate="label" sortOrder="200">
            <label>Dss</label>
        </tab>
        
        we used core module tab